gateway/src/api/latest.rs

```diff
@@ -307,7 +307,7 @@ async fn destroy_project(
 #[derive(Deserialize, IntoParams)]
 struct DeleteProjectParams {
     // Was added in v0.30.0
-    // We have not needed it since 0.35.0, but have to keep in for any old CLI users
+    // We have not needed it since 0.35.1, but have to keep in for any old CLI users
```

Comment:
Woops, we incorrectly bumped this last time.
```suggestion
    // We have not needed it since 0.34.1, but have to keep in for any old CLI users
```